tests/std/tests/VSO_0180469_ptr_cat/test.compile.pass.cpp

```diff
@@ -393,6 +393,8 @@ void equal_safe_test_cases() {
     test_case_Equal_memcmp_is_safe<false, base_class, base_class>();
 
 #ifdef __cpp_lib_byte
+    // GH-2039
+    static_assert(_Is_trivially_swappable_v<byte>);

```

Comment:
Should we add some more test for `_Is_trivially_swappable_v` (not only for `byte`)? I couldn't find any.